### ISSUE TITLE
Fix deposits on the testnet

### ISSUE THREAD

**Issue by troggy** [leapdao/meta#205]
**Problem:**
Since deposit 156, no deposits are not created on a testnet.

**Reason:**
Deposit 157 was a NFT token with value of "0", which wasn't processed by the node due to the deposit output validation error:

```
Dec 20 15:59:53 ip-172-31-31-90 leap-node[26799]: unhandledRejection Error: Output value is < 1
Dec 20 15:59:53 ip-172-31-31-90 leap-node[26799]:     at new Output (/usr/local/share/.config/yarn/global/node_modules/leap-core/dist/output.js:66:13)
Dec 20 15:59:53 ip-172-31-31-90 leap-node[26799]:     at Function.deposit (/usr/local/share/.config/yarn/global/node_modules/leap-core/dist/transaction.js:571:61)
Dec 20 15:59:53 ip-172-31-31-90 leap-node[26799]:     at Object.NewDeposit (/usr/local/share/.config/yarn/global/node_modules/leap-node/src/eventsRelay.js:63:23)
```

Since then all the deposits are rejected with

```
2020-01-13T13:54:30.380Z leap-node:error Deposit ID skipping ahead. want 157, found 158
Jan 13 13:55:33 ip-172-31-40-151 leap-node[25849]: 2020-01-13T13:55:33.411Z leap-node:error Deposit ID skipping ahead. want 157, found 159
```

**Resolution:**
- [x] fix leap-core to allow zero values for NFTs and NSTs. Fixed in leapdao/leap-core#169
- [x] verify the fix on a readonly node (5.4.1 with patch fix applied)
- [ ] release leap-node 6.0.0 with fixed leap-core (only that fix on top of 5.4.1, nothing else from master)
- [ ] upgrade testnet nodes to use leap-node 6.0.0. Will require coordinated update for both validators

**Comment by johannbarbie** [comment]
just out of interest, why 5.4 and not 5.5?

**Comment by troggy** [comment]
because it is just a bug fix?

**Comment by troggy** [comment]
I'm not sure how to apply semver rule here. Technically, 5.4.1 and patched version will process 0 NFTs differently and won't be compatible with each other, so this probably means the version should be 6.0.0 :)

**Comment by johannbarbie** [comment]
a small bug for consensus, a big leap for semver 🤣

**Comment by troggy** [comment]
fix released in leap-node@6.0.0 and is now live on testnet
